lib/ansible/playbook/base.py

```diff
@@ -386,13 +386,13 @@ def _resolve_group(self, fq_group_name, mandatory=True):
         return fq_group_name, resolved_actions
 
     def _resolve_action(self, action_name, mandatory=True):
-        context = module_loader.find_plugin_with_context(action_name)
+        context = module_loader.find_plugin_with_context(action_name, ignore_deprecated=not mandatory)
```

Comment:
just to keep people from updating this in a confusing way, i would use parens:
``` ignore_deprecated=(not mandatory))```
but not a show stopper.

Reply:
Fixed